hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/placement/csmappingrule/TestCSMappingPlacementRule.java

```diff
@@ -65,6 +70,11 @@ public class TestCSMappingPlacementRule {
 
   @Rule
   public TemporaryFolder folder = new TemporaryFolder();
+  
+  @Before
+  public void setUp() {
+    setLoggerToDebug();
```

Comment:
Did you purposely left this here?

Reply:
yep, I wanted to have a more verbouse output and I think it's not bad to keep the debug logs for later.
Is there any better way to increase verbosity of the logger? :)